tests/countries/test_turkmenistan.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+from datetime import date
+from holidays.countries import Turkmenistan
+
+class TestTurkmenistanHolidays(unittest.TestCase):
+    def setUp(self):
+        """Initialize Turkmenistan holidays for testing."""
+        self.holidays_2024 = Turkmenistan(years=2024)
+        self.holidays_2017 = Turkmenistan(years=2017)  
+        self.holidays_1991 = Turkmenistan(years=1991)  
+
+    def test_fixed_holidays_2024(self):
```

Comment:
Consider looking up [`test_kazakhstan.py`](https://github.com/vacanza/holidays/blob/f55391b28485f3dc427306ac214086603fcb014c/tests/countries/test_kazakhstan.py#L19) and [`test_macau.py`](https://github.com/vacanza/holidays/blob/dev/tests/countries/test_macau.py#L20) for examples